hathorlib/conf/settings.py

```diff
@@ -47,3 +47,6 @@ class HathorSettings(NamedTuple):
 
     # Maximum size of the tx output's script allowed for a tx to be standard
     PUSHTX_MAX_OUTPUT_SCRIPT_SIZE: int = 256
+
+    # Maximum number of tx outputs of Data Script type
+    MAX_DATA_SCRIPT_OUTPUTS: int = 25
```

Comment:
Is there any possibility that a non-nft transaction using a data script already exists in the Testnet or Mainnet with more than 25 outputs?

If such transaction exists, would we have any problem because of that?

Reply:
The standard that we create here in the lib is just for our public tx mining service and full nodes accept mining/pushing those transactions. The protocol does not have this limit, so there is no problem if we already have a tx that is non-standard.